### PR TITLE
Prepare 2.4.4

### PR DESCRIPTION
**Pull request by davisagli**
# Critical Changes

# Changes

# Issues Closed

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,22 @@
 History
 =======
 
+2.4.4 (2019-05-09)
+------------------
+
+New features:
+
+* Added tasks ``list_changes`` and ``retrieve_changes`` which interact with source tracking in scratch orgs to handle retrieving changed metadata as Metadata API format source.
+* Added task ``EnsureRecordTypes`` to generate a Record Type and optional Business Process for a specific sObject and deploy the metadata, if the record type does not already exist.
```
Comment (davidmreed) [inline HISTORY.rst]: How about "if the object does not already have Record Types"?

File: cumulusci/__init__.py [hunk 2]
```diff
@@ -2,5 +2,5 @@
 import os
 
 __import__("pkg_resources").declare_namespace("cumulusci")
-__version__ = "2.4.3"
+__version__ = "4"
```
Comment (davidmreed) [inline cumulusci/__init__.py]: Is this change intended? Shouldn't we have "2.4.4"?

Comment (davisagli) [reply cumulusci/__init__.py]: Ouch, yeah, this is a typo.
